src/main/java/com/exasol/adapter/dynamodb/documentpath/ArrayLookupPathSegment.java

```diff
@@ -0,0 +1,31 @@
+package com.exasol.adapter.dynamodb.documentpath;
+
+/**
+ * This path segment defines an array lookup in an path expression. E.g topics[0]
```

Comment:
Change as above. Or make the it as single sentence, `... path expression, e.g, {@code topics[0]}.`